config/environments/test.rb

```diff
@@ -20,6 +20,9 @@
     'Cache-Control' => 'public, max-age=3600',
   }
 
+  # Ensure that our tests are accounting for caching behaviour
+  config.cache_store = :memory_store
```

Comment:
I was just wondering how much memory the cache would take up, how big it might get if we cache a lot of these kinds of request. Default size for the cache is 32MB according to the docs so I guess it's not going to get excessive.